app/src/main/java/com/arygm/quickfix/ui/uiMode/appContentUI/userModeUI/search/SearchWorkerProfileResult.kt

```diff
@@ -50,10 +52,22 @@ fun SearchWorkerProfileResult(
     distance: Int? = null
 ) {
   val displayLocation =
-      if (location.length <= 10) {
+      if (location.length <= 9) {
         location
       } else {
-        location.take(10) + "..."
+        location.take(9) + "..."
+      }
+  val displayName =
+      if (name.length <= 25) {
+        name
+      } else {
+        name.take(25) + "..."
+      }
+  val displayCateg =
+      if (category.length <= 21) {
+        category
+      } else {
+        category.take(21) + "..."
       }
```

Comment:
It might be better to create some constants here explaining the choice of these particular values.